docs/content/en/docs-v1.0.x/user-guide/managing-application/deployment-chain.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Deployment chain"
+linkTitle: "Deployment chain"
+weight: 11
+description: >
+  Specific guide for configuring chain of deployments.
+---
+
+For users who want to use PipeCD to build a complex deployment flow, which contains multiple applications across multiple application kinds and roll them out to multiple clusters gradually or promoting across environments, this page will show you how to use PipeCD to achieve that requirement.
+
+## Configuration
+
+The idea of this feature is to trigger the whole deployment chain when a specified deployment is triggered. To trigger the deployment chain, we need to add a configuration section named `postSync` which contains all configurations that be used when the deployment is triggered. For this `Deployment Chain` feature, configuration for it is under `postSync.chain` section.
+
+A canonical configuration looks as below:
+
+```yaml
+apiVersion: pipecd.dev/v1beta1
+kind: Application
+spec:
+  input:
+    ...
+  pipeline:
+    ...
+  postSync:
+    chain:
+      applications:
+        # Find all applications with name `application-2` and trigger them.
+        - name: application-2
+        # Fill all applications with name `application-3` of kind `KUBERNETES`
+        # and trigger them.
+        - name: application-3
+          kind: KUBERNETES
+```
+
+As a result, the above configuration will be used to create a deployment chain like the below figure
+
+![Screenshot of deployment chain feature](/images/deployment-chain-figure.png)
+
+In the context of the deployment chain in PipeCD, a chain is made up of many `blocks`, and each block contains multiple `nodes` which is the reference to a deployment. The first block in the chain always contains only one node, which is the deployment that triggers the whole chain. Other blocks of the chain are built using filters which are configurable via `postSync.chain.applications` section. As for the above example, the second block `Block 2` contains 2 different nodes, which are 2 different PipeCD applications with the same name `application-2`.
+
+__Tip__:
+
+1. If you followed all the configuration references and built your deployment chain configuration, but some deployments in your defined chain are not triggered as you want, please re-check those deployments [`trigger configuration`](../triggering-a-deployment/#trigger-configuration). The `onChain` trigger is __disabled by default__; you need to enable that configuration to enable your deployment to be triggered as a node in the deployment chain.
+2. Values configured under `postSync.chain.applications` - we call it __Application matcher__'s values are merged using `AND` operator. Currently, only `name` and `kind` are supported, but `labels` will also be supported soon.
+
+See [Examples](../../examples/#deployment-chain) for more specific.
```

Comment:
We do not have the examples directory for v1 right now, should remove this link to avoid 404 redirect